general-information.md

```diff
@@ -4,32 +4,39 @@ This document contains specifications that are shared between the various MDS AP
 
 ## Table of Contents
 
-* [Beta Features](#beta-features)
-* [Costs and Currencies](#costs-and-currencies)
-* [Definitions](#definitions)
-* [Devices](#devices)
-* [Geographic Data][geo]
-  * [Geographic Telemetry Data](#geographic-telemetry-data)
-  * [Stop-based Geographic Data](#stop-based-geographic-data)
-  * [Intersection Operation](#intersection-operation)
-* [Geography-Driven Events](#geography-driven-events)
-* [Optional Authentication](#optional-authentication)
-* [Propulsion Types](#propulsion-types)
-* [Responses](#responses)
-  * [Error Messages](#error-messages)
-* [Strings](#strings)
-* [Stops](#stops)
-  * [Stop Status](#stop-status)
-  * [GBFS Compatibility](#gbfs-compatibility)
-* [Timestamps](#timestamps)
-* [UUIDs](#uuids)
-* [Vehicle States](#vehicle-states)
-  * [Event Types](#event-types)
-  * [Limitations on the Use of Certain Values](#limitations-on-the-use-of-certain-values)
-  * [Vehicle State Events](#vehicle-state-events)
-  * [State Machine Diagram](#state-machine-diagram)
-* [Vehicle Types](#vehicle-types)
-* [Versioning](#versioning)
+- [Mobility Data Specification: **General information**](#mobility-data-specification-general-information)
```

Comment:
Flagging for minor update to the TOC formats in the PR markdown to match what's there now? Eg, one level shallower, '-' character, alphabetized in general information, on home page, etc.

Reply:
This was all done by the auto-formatter I have installed in my VSCode, it'd be nice if we could standardize on a formatting _library_ to do these things (could check in a .vscode with a recommended extension or something). Updating TOC shouldn't be something anyone has to do manually!

Reply:
Agree! Here are the "rules" across the spec now:
1. don't include the page title
3. use '-' instead of '*'
4. record 3 header level in the toc

Reply:
👍🏼